src/metabase/analytics/prometheus.clj

```diff
@@ -232,18 +232,26 @@
    (prometheus/counter :metabase-search/response-error
                        {:description "Number of errors when responding to search requests."})])
 
+(defmulti known-labels
+  "Implement this for a given metric to initialize zeros for the given set of label values."
+  identity)
```

Comment:
It would be much nicer if we could just pass the labels in at the time we define the counter. Unfortunately the way Iapetos works is that its "collector" are actually factories, and the *real* collectors are only created later, when we register them in `setup-metrics!`.

The upside of this structure is that we don't create a bi-directional dependency on the search module.

It would be nice if we could force a start-up error if no labels are provided for a labelled metric, but I didn't see a simple way to do that, and didn't want to add further complexity for now.